docs/platform/connector-development/config-based/understanding-the-yaml-file/property-chunking.md

```diff
@@ -0,0 +1,309 @@
+# Property chunking
+
+Property chunking enables connectors to handle APIs with limitations on the number of properties that you can fetch per request. This feature breaks down large property lists into smaller, manageable chunks and merges the results back into complete records. Several connectors require this capability to work with APIs that have property limits.
+
+## Overview
+
+Property chunking works in these steps.
+
+1. Fetching the complete list of properties (either statically defined or dynamically from an API endpoint)
+
+2. Splitting properties into chunks based on configured limits
+
+3. Making separate API requests for each chunk
+
+4. Merging the results back into complete records using a merge strategy
```

Comment:
maybe just provide an example of a merge strategy, like combing sets of property values into a single record according to their primary key id